src/registerToolbar.ts

```diff
@@ -70,6 +72,24 @@ export default function registerToolbar() {
         type: ToolbarItemType.Action,
     });
 
+    ToolbarItemFactory.register(PreviousAsset, {
+        name: "navigatePreviousAsset",
+        tooltip: "Previous Asset",
```

Comment:
good for now, but we should pipe localization in here.

Reply:
👍